timemachine/fe/rbfe.py

```diff
@@ -876,15 +880,19 @@ def run_solvent(
     md_params: MDParams = DEFAULT_HREX_PARAMS,
     n_windows: Optional[int] = None,
     min_overlap: Optional[float] = None,
-    min_cutoff: Optional[float] = 0.7,
+    min_cutoff: Optional[float] = None,
 ):
     if md_params is not None and md_params.water_sampling_params is not None:
         md_params = replace(md_params, water_sampling_params=None)
         warnings.warn("Solvent simulations don't benefit from water sampling, disabling")
     box_width = 4.0
-    solvent_sys, solvent_conf, solvent_box, solvent_top = builders.build_water_system(box_width, forcefield.water_ff)
+    solvent_sys, solvent_conf, solvent_box, solvent_top = builders.build_water_system(
+        box_width, forcefield.water_ff, mols=[mol_a, mol_b]
+    )
     solvent_box += np.diag([0.1, 0.1, 0.1])  # remove any possible clashes, deboggle later
     solvent_host_config = HostConfig(solvent_sys, solvent_conf, solvent_box, solvent_conf.shape[0])
```

Comment:
nit, maybe future PR: would it be useful to move this into a reusable `get_solvent_host_config`?